ivy/functional/backends/paddle/manipulation.py

```diff
@@ -69,6 +73,10 @@ def expand_dims(
     return x.reshape(out_shape)
 
 
+@with_unsupported_dtypes(
+    {"2.5.1 and below": ("bfloat16", "float16", "int16", "int8", "uint8")},
```

Comment:
I assume you've already verified that `bfloat16` isn't supported for this function? Same applies to the other functions where `bfloat16 ` was marked as unsupported

Reply:
yes I verified `bfloat16` here and it's not supported.
my approach for all functions modified here is to loop through every `dtype `in `paddle ` and call the function with this `dtype` and if there's not error or strange behavior( like input dtype not the same as output dtype) then I'll mark the dtype as supported.
Hope that make sense 😀

Reply:
Yep makes perfect sense, thanks!